client/app/lib/statefulForm.js

```diff
@@ -0,0 +1,127 @@
+/** @jsx h */
+import { h, Component } from 'preact'
```

Comment:
Yes, it's more of a helper, but still it contains JSX... Not sure what to do here...